README.rst

```diff
@@ -127,16 +127,16 @@ We strongly encourage you to comment on our roadmap and provide us feedback on t
 GitHub issue.
 
 .. |dali-roadmap-link| replace:: The following issue represents
-.. _dali-roadmap-link: https://github.com/NVIDIA/DALI/issues/3774
+.. _dali-roadmap-link: https://github.com/NVIDIA/DALI/issues/4578
 
 ----
 
 Installing DALI
 ---------------
 
-To install the latest DALI release for the latest CUDA version (11.x)::
+To install the latest DALI release for the latest CUDA version (12.x)::
 
-    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist --upgrade nvidia-dali-cuda110
+    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist --upgrade nvidia-dali-cuda120
```

Comment:
How about putting the information about necessary CUDA install also here?
![image](https://user-images.githubusercontent.com/7101064/215064900-f89407e1-4fc1-4ceb-9896-3bbed476c572.png)
So that when somebody goes to the README, pastes these instructions in "installing DALI", it works?

Reply:
Do you have some wording to suggest? I think the problem is, that the CUDA toolkit may be installed in several different ways depending on the system and you shouldn't mix those if you already used one of installation paths already.
Especially if you have toolkit, you don't want to repeat the install by copy-pasting it.

Reply:
Generally, right now user needs to know, that he needs to install CUDA first and then DALI. Anything that would suffice it works.

How about something along these lines:
```
A first step to installing DALI is to install CUDA toolkit. There are many possible ways of installing it, e.g.:
1. [Using `pip install`](https://nvidia.github.io/cuda-python/install.html#installing-from-pypi): `pip install cuda-python`
1. [Using runfile](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu&target_version=22.04&target_type=runfile_local) 
1. [Installing from conda](https://nvidia.github.io/cuda-python/install.html#installing-from-conda)

If the CUDA has been installed previously, it is not necessary to reinstall using any of these methods.

To install the latest DALI release [...]
```

Reply:
It is already in requirements, linked to the CUDA Toolkit installation guide https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html
which covers the topic extensively. I don't think it is good idea to copy a small subset of this page here.
Typically users working on DL will already have the CUDA installed for the purposes of the framework.
We don't specify how to install any of the frameworks or the driver either.